src/test/java/jerry/codecamp/DogTest.java

```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
     }
 
     @Test
-    public void timePassesForDog_increasesThirstLevel(){
+    public void timePassesForDog_increasesThirstLevel() {
         dog.timePassesForDog();
         assertEquals(5, dog.getThirstLevel());
 
         dog.timePassesForDog();
         assertEquals(10, dog.getThirstLevel());
     }
 
+    @Test
+    public void giveTheDogASnack_increasesHungerLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(- 10, dog.getHungerLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getHungerLevel());
+
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesLonelyLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(-10, dog.getLonelyLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getLonelyLevel());
+    }
```

Comment:
Giving a snack increases or decreases?

Change to decreases.

Either do something that makes the LonelyLevel Go UP or make a setter and set the value to something > 20.